### ISSUE TITLE
Name & Logo Design!

### ISSUE THREAD

**Issue by white105** [white105/codecomp#9]
Codecomp isn't really set in stone. It's still pretty much a concept at this point. Would love to hear more ideas for names!

The logo also needs to redone because it's a little bit blurry. Since we're not using trophies anymore maybe we can change the logo all together

**Comment by busi-reddy-karnati** [comment]
Any thoughts as to how the logo should look like?

**Comment by white105** [comment]
@busi-reddy-karnati Not really! Pretty much anything that looks cool. I could potentially ask a few people on fivver to try and make a video out of it too

**Comment by white105** [comment]
This is going to be a fun platform though with memes and stuff (Twitch Like) so we wouldn't want to aim for anything too prestigious

**Comment by busi-reddy-karnati** [comment]
How about the name to be codebase

**Comment by busi-reddy-karnati** [comment]
![Screenshot (21)](https://user-images.githubusercontent.com/31757507/71776697-115ebd00-2fbc-11ea-8d31-004ffbffc056.png)

**Comment by busi-reddy-karnati** [comment]
How about this one
I know it is not the best, but we have to start with something

**Comment by white105** [comment]
That'a pretty cool! I'd like to stick to a more purple color but other than that i like it!

**Comment by busi-reddy-karnati** [comment]
Sure. I will change and update.

**Comment by busi-reddy-karnati** [comment]
The thunderbolt on the left symbolizes a raw idea(brain) and the one on right symbolizes a fully formed one indicating a developed brain. This tells what our platform does. Just in case if you are wondering why that.
